mindmaps-graph/src/main/java/io/mindmaps/graph/internal/AbstractMindmapsGraph.java

```diff
@@ -364,6 +364,11 @@ public Instance getInstance(String id) {
     }
 
     @Override
+    public Instance getInstance(Resource uniqueResource){
+        return uniqueResource.owner();
+    }
```

Comment:
Ya I am not sure. I am in favour of removing it but the requirement asked for a get based on resources.